src/sider/oppfolgingsplan/oppfolgingsplaner/BeOmOppfolgingsplan.tsx

```diff
@@ -0,0 +1,92 @@
+import {
+  Alert,
+  BodyLong,
+  BodyShort,
+  Box,
+  Button,
+  Heading,
+} from "@navikt/ds-react";
+import React from "react";
+import {
+  NewOppfolgingsplanForesporselDTO,
+  useGetOppfolgingsplanForesporselQuery,
+  usePostOppfolgingsplanForesporselQuery,
+} from "@/data/oppfolgingsplan/oppfolgingsplanQueryHooks";
+import { useValgtPersonident } from "@/hooks/useValgtBruker";
+import { Veileder } from "@/data/veilederinfo/types/Veileder";
+import { NarmesteLederRelasjonDTO } from "@/data/leder/ledereTypes";
+import { OppfolgingstilfelleDTO } from "@/data/oppfolgingstilfelle/person/types/OppfolgingstilfellePersonDTO";
+import { tilLesbarDatoMedArUtenManedNavn } from "@/utils/datoUtils";
+
+const texts = {
+  header: "Be om oppfølgingsplan fra arbeidsgiver",
+  description:
+    "Her kan du be om oppfølgingsplan fra arbeidsgiver eller purre om det mangler.",
+  virksomhet: "Virksomhet:",
+  narmesteLeder: "Nærmeste leder:",
+  aktivForesporsel:
+    "Obs! Det ble bedt om oppfølgingsplan fra denne arbeidsgiveren",
+  button: "Be om oppfølgingsplan",
+};
+
+interface Props {
+  aktivVeileder: Veileder;
+  activeNarmesteLeder: NarmesteLederRelasjonDTO;
+  latestOppfolgingstilfelle: OppfolgingstilfelleDTO;
+}
+
+export default function BeOmOppfolgingsplan({
+  aktivVeileder,
+  activeNarmesteLeder,
+  latestOppfolgingstilfelle,
+}: Props) {
+  const personident = useValgtPersonident();
+  const getOppfolgingsplanForesporsel = useGetOppfolgingsplanForesporselQuery();
+
+  const postOppfolgingsplanForesporselQuery =
+    usePostOppfolgingsplanForesporselQuery();
+
+  function onClick() {
+    const foresporsel: NewOppfolgingsplanForesporselDTO = {
+      arbeidstakerPersonident: personident,
+      veilederident: aktivVeileder.ident,
+      virksomhetsnummer: activeNarmesteLeder.virksomhetsnummer,
+      narmestelederPersonident:
+        activeNarmesteLeder.narmesteLederPersonIdentNumber,
+    };
+    postOppfolgingsplanForesporselQuery.mutate(foresporsel);
+  }
+
+  const lastForesporselCreatedAt =
+    getOppfolgingsplanForesporsel.data?.createdAt;
+
+  const isActiveForesporsel = !!lastForesporselCreatedAt
+    ? latestOppfolgingstilfelle.start <= lastForesporselCreatedAt &&
+      lastForesporselCreatedAt <= latestOppfolgingstilfelle.end
+    : false;
```

Comment:
Blir dette riktig? Om vi er innenfor samme oppfølgingstilfelle skal det vel være en aktiv forespørsel? 🤔

Reply:
Synes det virker logisk. Og også kanskje at man bare kan be om oppfølgingsplan én gang for et oppfølgingstilfelle? Men det må sikkert avklares.